code/datums/emergency_calls/kelion.dm

```diff
@@ -0,0 +1,54 @@
+/datum/emergency_call/kellionteam
+	name = "Kellionteam"
+	probability = 50
+	landmark_tag = "kellionteam"
+	specials_outfits = list(/decl/hierarchy/outfit/isaac, /decl/hierarchy/outfit/kendra)
+
+/datum/emergency_call/kellionteam/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<B>You are Kellionteam.</b>")
+	to_chat(H, "")
+	to_chat(H, "<B>Follow any orders directly from Kellionteam Central Command.</b>")
+
+/datum/emergency_call/kellionteam/create_member(datum/mind/M)
+	. = ..()
+	if(!.)
+		return
+
+	var/mob/original = M.current
+	var/mob/living/carbon/human/H = .
+
+	H.name = pick(GLOB.first_names_female + GLOB.first_names_male) + " " + pick(GLOB.last_names)
+	H.real_name = H.name
+
+	M.transfer_to(H, TRUE)
+	H.fully_replace_character_name(M.name, H.real_name)
+
+	if(original)
+		qdel(original)
+
+	print_backstory(H)
+
+	var/decl/hierarchy/outfit/ertfit = new /decl/hierarchy/outfit/kellion_sec
+	if(!leader)
+		leader = H
+		ertfit = new /decl/hierarchy/outfit/kellion_sec_leader
+		dressup_human(H, ertfit)
+		to_chat(H, "<p style='font-size:1.5em'><span class='notice'>You are the leader of squad.</span></p>")
+		return
+
+	if(specials_outfits)
```

Comment:
It really doesn't sound like a good idea to give someone such famous names

Reply:
There are no variables that use this abbreviation. So it's good to call things own things.

Reply:
what famous names, you mean kellion? That was already planned long ago, these ERTs existed in code before this PR